### PR TITLE
Name a park flag and a TD6 flag.

### PR DESCRIPTION
**Pull request by Gymnasiast**
The latter as requested by Duncan

### CODE REVIEW

File: src/ride/track.h [hunk 1]
```diff
@@ -173,7 +173,7 @@ typedef struct {
 	uint8 track_spine_colour[4];					// 0x60
 	uint8 track_rail_colour[4];						// 0x64
 	uint8 track_support_colour[4];					// 0x68
-	uint32 var_6C;									// some flags
+	uint32 var_6C;									// some extra flags. (1 << 31) was used to mark the TD6 as a Six Flags ride.
```
Comment (IntelOrca) [inline src/ride/track.h]: Perhaps rename this to flags2 and use a new enum for the SIX_FLAGS flag.

Comment (Gymnasiast) [inline src/ride/track.h]: Do we really need to make an enum for a flag we don't use anymore? Duncan only asked to clarify it in a comment, that's why I put it here.

Renaming var_6C to flags2 would be a good idea, I think.

Comment (IntelOrca) [inline src/ride/track.h]: Aren't there other flags?

Comment (Gymnasiast) [inline src/ride/track.h]: There are. Ah well, I'll put the deprecated Six Flags flag in the enum anyway.

Comment (IntelOrca) [inline src/ride/track.h]: Its still good to have it for future reference though,

**Pull request merged**